lib/elixir/lib/record.ex

```diff
@@ -298,6 +298,15 @@ defmodule Record do
   defp create(atom, fields, keyword, caller) do
     in_match = Macro.Env.in_match?(caller)
 
+    #backward compatiblity for Erlang, a :_ key sets all fields to a default value
+    {fields, keyword} = case Dict.pop(keyword, :_) do
```

Comment:
As you said in the PR, `Dict` is deprecated. `Keyword.pop` should do the job here.